api/requests.js

```diff
@@ -0,0 +1,27 @@
+const RequestHeader = {
+  'Content-type': 'application/json; charset=UTF-8',
+};
+
+export const getData = async () => {
+  try {
+    const response = await fetch('http://localhost:4000/countries', {
+      method: 'GET',
+      headers: RequestHeader,
+    });
+    return response.json();
+  } catch (e) {
+    alert('Fail to get data from server. Try again later!');
+  }
+};
+
+export const changeData = async (rowId, changedDataObj) => {
+  try {
+    await fetch(`http://localhost:4000/countries/${rowId}`, {
+      method: 'PUT',
+      headers: RequestHeader,
+      body: JSON.stringify(changedDataObj),
```

Comment:
I'm not sure, that we need to stringify the body
It should work with an object also